crates/cli-support/src/wit/mod.rs

```diff
@@ -137,13 +137,13 @@ impl<'a> Context<'a> {
         }
         self.add_aux_import_to_import_map(
             "__wbindgen_object_clone_ref",
-            vec![Descriptor::I32],
-            Descriptor::I32,
+            vec![Descriptor::Ref(Box::new(Descriptor::Externref))],
```

Comment:
Hm our original intrinsics were `u32` though, shouldn't externref pass be the one that changes them to `externref`?

Otherwise I'd think we're breaking the non-externref target.

Reply:
Hm probably you're right and I got confused, I guess I thought this has to be the raw signature here to match the original `fn __wbindgen_object_drop_ref(idx: u32) -> ();`.

I still don't get how integration tests haven't caught this, surely they use `__wbindgen_object_drop_ref` a lot.

Reply:
Nvm, saw the discussion below, let's continue there.